lib/ApiClient.js

```diff
@@ -9,15 +9,19 @@ class ApiClient extends AxiosClient {
     autobind(this);
   }
 
-  getUser(jwtToken, params = {}) {
-    const path = 'account';
```

Comment:
We could set the path parameter right here, so we wouldn't need those `if {} else {}`. Like:
`const path = id ? 'drops/${id}' : 'account;'`
😉 
@rafalswietek what do you think?

Reply:
@aliaksandr-savitski Generally speaking I'm against the changes proposed here.
This function right now is getting user with specified jwtToken and passed params so adding additional param may be nice way to improve the functionality but order of params is being inconsistent with rest of the functions.
To keep proper order it should be `getUser(id = null, jwtToken, params = {})` and by doing so we will make it backward incompatible.

My suggestion is to introduce new lexical method `getUserById` with above setup of params and it should do the trick :)

Reply:
Guys, regarding @Necromos opinion, I will apply this solution. Let me know If it suits you :)

Reply:
I've been thinking about this solution too, but it seemed to me like a non consistent method name according to the rest of this SDK.
Although I've seen worst in aws-sdk so I'm ok with it 👍